static/src/javascripts/projects/common/modules/email/email.js

```diff
@@ -262,7 +262,8 @@ define([
                 return function (event) {
                     var emailAddress = $('.' + classes.textInput, $form).val(),
                         listId = $('.' + classes.listIdHiddenInput, $form).val(),
-                        markCheckbox = $('.' + classes.markCheckbox, $form)[0].checked ? 'marketing allowed' : 'marketing disallowed',
+                        $markCheckbox = $('.' + classes.markCheckbox, $form),
+                        markCheckbox = $markCheckbox.length && $markCheckbox[0].checked ? 'marketing allowed' : 'marketing disallowed',
```

Comment:
Was it a `Cannot read property 'foo' of undefined`? :tongue: 

We could use Options to avoid this class of errors.